test/SlowTests/Server/Documents/PeriodicBackup/Retention.cs

```diff
@@ -31,7 +31,7 @@ public async Task can_delete_backups_by_date(int backupAgeInSeconds, int numberO
             {
                 BackupConfigurationHelper.SkipMinimumBackupAgeToKeepValidation = true;
 
-                var backupPath = NewDataPath(suffix: "BackupFolder");
+                var backupPath = NewDataPath(suffix: "BackupFolder") + "/Backups/Admin\\..\\..";
```

Comment:
Pretty sure that this won't work on Linux, no?

Reply:
Should work. `PathUtil` will normalize it